### PR TITLE
Mcs 1539 toggle metadata

### PR DESCRIPTION
**Pull request by phile-caci**
Here's the AI2THOR portion to handle the metadata toggle for disable_object_list and disable_position.

### CODE REVIEW

**Comment by rartiss55** [comment]
Do the depth maps and object masks need to be incorporated here as well?

**Comment by phile-caci** [comment]
> Do the depth maps and object masks need to be incorporated here as well?

Hi @rartiss55 ,

I don't think so, they were already toggled from MCS but just at a metadata tier level. Their parameters already existed. In MCS I made it toggle with the metadata tier and the new config parameters.

from machine_common_sense/parameter.py
111            renderDepthImage=self.config.is_depth_maps_enabled(),
112            renderObjectImage=self.config.is_object_masks_enabled(),

I updated config.is_depth_maps_enabled() and config.is_object_masks_enabled() so that the toggle took into account the metadata tier and the new config parameters.

Let me know if I missed anything, thanks!

**Review by rartiss55 — APPROVED** [review]

File: unity/Assets/Scripts/BaseFPSAgentController.cs [hunk 1]
```diff
@@ -1587,11 +1588,13 @@ public virtual MetadataWrapper generateMetadataWrapper()
...
-            if (this.disableObjectList)
-                metaMessage.objects = Array.Empty<ObjectMetadata>();
-            else
-                metaMessage.objects = this.generateObjectMetadata();
-
+            // if (this.onlyReturnObjectGoal)
+            //     // TODO: update generateObjectMetadata to only return goal objects
+            //     metaMessage.objects = this.generateObjectMetadata();
+            // else
+            //     metaMessage.objects = this.generateObjectMetadata();
```
Comment (ThomasSchellenbergNextCentury) [inline unity/Assets/Scripts/BaseFPSAgentController.cs:1595]: Is this still here for a reason, or should it be deleted?

Comment (phile-caci) [reply unity/Assets/Scripts/BaseFPSAgentController.cs]: Sry, missed it.  It's removed now.

**Pull request merged**